packages/protocol/contracts/L1/TaikoConfig.sol

```diff
@@ -34,20 +36,17 @@ library TaikoConfig {
                 slotSmoothingFactor: 946649,
                 // 100 basis points or 1%
                 rewardBurnBips: 100,
-                proposerDepositPctg: 25, // - 25%
                 // Moving average factors
                 feeBaseMAF: 1024,
-                bootstrapDiscountHalvingPeriod: 1 seconds, // owner:daniel
-                constantFeeRewardBlocks: 1024,
                 txListCacheExpiry: 0,
+                proofTimeTarget: 85, // 85s based on A2 testnet status
```

Comment:
Ok, wee need to keep that in mind, but I would leave it as is for the next couple iterations (testnets), because if:
 `proofTime` highly differs from `proofTimeTarget` AND `allowMinting` is `true` we will not reach balance and deposits will be much bigger than withdrawals - which is net good, but definitely an un-balanced state.